src/Model/UserManager.php

```diff
@@ -4,7 +4,9 @@
 
 class UserManager extends AbstractManager
 {
-    public const TABLE = 'user';
+    public const TABLE = 'comic_book';
+    public const TABLE2 = 'contact';
+    public const TABLE3 = 'user';
```

Comment:
No why the main TABLE of your UserManager is "comic_book" ??? It should be and stay "user"